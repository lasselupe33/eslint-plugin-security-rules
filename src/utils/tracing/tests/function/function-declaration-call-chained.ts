function a(x: number) {
  return "I have " + x + " cookies!";
}

function b(n: number) {
  return a(n);
}

function c(z: number) {
  return b(z);
}

const start = c(10);

export {};
