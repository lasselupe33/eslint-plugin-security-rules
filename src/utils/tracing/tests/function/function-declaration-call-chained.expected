I have 10 cookies!